src/rules/sequenceValidator.js

```
@@ -0,0 +1,67 @@
+      type: "sequence_length_error",
+      head: structure.heading.content,
+      message: `Expected ${template.sequence.length} content types in sequence, but found ${structure.content.length}`,
+      position: structure.position,
+    });
+    return errors;
+  }
+
+  // Check sequence order
+  const templateItemTypes = template.sequence.map(item => Object.keys(item)[0]);
+  const structureItemTypes = structure.content.map(item => {
+    if (item.hasOwnProperty("paragraphs")) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)